src/main/java/assets/game.js

```diff
@@ -318,6 +335,38 @@ function place(size) {
     }
 }
 
+function killTwoShips()
```

Comment:
Ooh, I see you must've added this for testing. I've got not problem with this, I played literally dozens and dozens of games just to test features I put in. Smart!